### DISCUSSION TITLE
Q&A: After Startup, webpage refuse the connection

### DISCUSSION THREAD

**Discussion by Timsto** [shlinkio/discussions#17] [Help wanted] [answered]
### shlink-web-client version

latest: 

### How do you use shlink-web-client

Docker image

### Summary

After startup (with no error) the container seems up and running but calling http://ip:port i got "connection refused" error
it appears in all browsers. 

Only difference from offical docu is that i use Port 8001 for publish. 

Any idea? 

This is the docker startup log: 

/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
10-listen-on-ipv6-by-default.sh: info: can not modify /etc/nginx/conf.d/default.conf (read-only file system?)
/docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-shlink-servers-json.sh
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
/docker-entrypoint.sh: Configuration complete; ready for start up
2024/02/11 22:00:58 [notice] 1#1: using the "epoll" event method
2024/02/11 22:00:58 [notice] 1#1: nginx/1.25.3
2024/02/11 22:00:58 [notice] 1#1: built by gcc 12.2.1 20220924 (Alpine 12.2.1_git20220924-r10) 
2024/02/11 22:00:58 [notice] 1#1: OS: Linux 5.15.0-91-generic
2024/02/11 22:00:58 [notice] 1#1: getrlimit(RLIMIT_NOFILE): 1048576:1048576
2024/02/11 22:00:58 [notice] 1#1: start worker processes
2024/02/11 22:00:58 [notice] 1#1: start worker process 22

**Comment by acelaya** [comment, marked as answer]
See https://github.com/shlinkio/shlink-web-client/issues/1052#issuecomment-1922397908
